### ISSUE TITLE
Generated `quickstart-web.xml` refers to jarfiles under Gcloud SDK installation directory

### ISSUE THREAD

**Issue by acourtneybrown** [GoogleCloudPlatform/app-gradle-plugin#71]
I am trying to make use of the Java quickstart functionality of Jetty & the Flexible VM compat runtime, by including the relevant configuration in the `appengine-web.xml` and adding 

``` groovy
stage {
  enableQuickstart = true
}
```

to the `build.gradle` file.

However, with this configuration added, there are a number of values in the generated `quickstart-web.xml` under `org.eclipse.jetty.tlds` referring to local jarfiles, which ultimately cause a deployment to GCP to fail.  For example, here's the relevant section from `build/staged-app/WEB-INF/quickstart-web.xml` from trying this with the [`getting-started-java/helloworld-compat`](https://github.com/GoogleCloudPlatform/getting-started-java/tree/master/helloworld-compat) sample project.

``` xml
  <context-param>
    <param-name>org.eclipse.jetty.tlds</param-name>
    <param-value><![CDATA[
    "jar:file:/Users/abrown/google-cloud-sdk/platform/google_appengine/google/appengine/tools/java/lib/java-managed-vm/appengine-java-vmruntime/lib/jsp/taglibs-standard-jstlel-1.2.3.jar!/META-INF/fmt-1_0.tld",
    "jar:file:/Users/abrown/google-cloud-sdk/platform/google_appengine/google/appengine/tools/java/lib/java-managed-vm/appengine-java-vmruntime/lib/jsp/taglibs-standard-impl-1.2.3.jar!/META-INF/sql.tld",
    "jar:file:/Users/abrown/google-cloud-sdk/platform/google_appengine/google/appengine/tools/java/lib/java-managed-vm/appengine-java-vmruntime/lib/jsp/taglibs-standard-impl-1.2.3.jar!/META-INF/x-1_0-rt.tld",
    "jar:file:/Users/abrown/google-cloud-sdk/platform/google_appengine/google/appengine/tools/java/lib/java-managed-vm/appengine-java-vmruntime/lib/jsp/taglibs-standard-jstlel-1.2.3.jar!/META-INF/c-1_0.tld",
    "jar:file:/Users/abrown/google-cloud-sdk/platform/google_appengine/google/appengine/tools/java/lib/java-managed-vm/appengine-java-vmruntime/lib/jsp/taglibs-standard-impl-1.2.3.jar!/META-INF/c-1_1.tld",
    "jar:file:/Users/abrown/google-cloud-sdk/platform/google_appengine/google/appengine/tools/java/lib/java-managed-vm/appengine-java-vmruntime/lib/jsp/taglibs-standard-impl-1.2.3.jar!/META-INF/scriptfree.tld",
    "jar:file:/Users/abrown/google-cloud-sdk/platform/google_appengine/google/appengine/tools/java/lib/java-managed-vm/appengine-java-vmruntime/lib/jsp/taglibs-standard-impl-1.2.3.jar!/META-INF/sql-1_0-rt.tld",
    "jar:file:/Users/abrown/google-cloud-sdk/platform/google_appengine/google/appengine/tools/java/lib/java-managed-vm/appengine-java-vmruntime/lib/jsp/taglibs-standard-impl-1.2.3.jar!/META-INF/fmt-1_0-rt.tld",
    "jar:file:/Users/abrown/google-cloud-sdk/platform/google_appengine/google/appengine/tools/java/lib/java-managed-vm/appengine-java-vmruntime/lib/jsp/taglibs-standard-impl-1.2.3.jar!/META-INF/fmt.tld",
    "jar:file:/Users/abrown/google-cloud-sdk/platform/google_appengine/google/appengine/tools/java/lib/java-managed-vm/appengine-java-vmruntime/lib/jsp/taglibs-standard-impl-1.2.3.jar!/META-INF/c.tld",
    "jar:file:/Users/abrown/google-cloud-sdk/platform/google_appengine/google/appengine/tools/java/lib/java-managed-vm/appengine-java-vmruntime/lib/jsp/taglibs-standard-impl-1.2.3.jar!/META-INF/c-1_0-rt.tld",
    "jar:file:/Users/abrown/google-cloud-sdk/platform/google_appengine/google/appengine/tools/java/lib/java-managed-vm/appengine-java-vmruntime/lib/jsp/taglibs-standard-impl-1.2.3.jar!/META-INF/permittedTaglibs.tld",
    "jar:file:/Users/abrown/google-cloud-sdk/platform/google_appengine/google/appengine/tools/java/lib/java-managed-vm/appengine-java-vmruntime/lib/jsp/taglibs-standard-impl-1.2.3.jar!/META-INF/fn.tld",
    "jar:file:/Users/abrown/google-cloud-sdk/platform/google_appengine/google/appengine/tools/java/lib/java-managed-vm/appengine-java-vmruntime/lib/jsp/taglibs-standard-impl-1.2.3.jar!/META-INF/x.tld",
    "jar:file:/Users/abrown/google-cloud-sdk/platform/google_appengine/google/appengine/tools/java/lib/java-managed-vm/appengine-java-vmruntime/lib/jsp/taglibs-standard-jstlel-1.2.3.jar!/META-INF/x-1_0.tld",
    "jar:file:/Users/abrown/google-cloud-sdk/platform/google_appengine/google/appengine/tools/java/lib/java-managed-vm/appengine-java-vmruntime/lib/jsp/taglibs-standard-jstlel-1.2.3.jar!/META-INF/sql-1_0.tld"]]></param-value>
  </context-param>
```

Without the changes to `build.gradle`, `quickstart-web.xml` does not get generated (and our deployment ultimately fails/times-out because Jetty (really, [GoogleCloudPlatform/appengine-java-vm-runtime](https://github.com/GoogleCloudPlatform/appengine-java-vm-runtime) insists on performing annotation scanning through the ~150 jarfiles in our webapp).

Seems like `appengine-gradle-plugin` is having issues with quickstart. Or, am I incorrectly using the plugin?

**Comment by meltsufin** [comment]
@ludoch is this an issue with staging?

**Comment by arouzrokh** [comment]
ping @ludoch

**Comment by patflynn** [comment]
@ludoch ping

**Comment by ludoch** [comment]
This was a Jetty regression https://github.com/eclipse/jetty.project/issues/1038 , that is now fix with the GAE Java SDK 1.9.48.
The fix should propagate to the Cloud SDK in the next 2 weeks.
